### DISCUSSION TITLE
I'm curious how solid compares to alpinejs

### DISCUSSION THREAD

**Discussion by jasikpark** [solidjs/solid#469] [Ideas]
They both claim to be a compile-free vdom-free reactivity library 🤔

**Comment by zhaoyao91** [comment]
A view from performance, alpinejs is the last but solidjs is nearly the first.

see https://krausest.github.io/js-framework-benchmark/current.html

**Comment by ryansolid** [comment]
Alpine has slightly different goals. Solid is mostly recommended to be used with JSX and compiled. While you can use it directly in the browser without compilation it is geared towards client rendering whereas Alpine is set up for augmenting previously server-rendered HTML Alpine inlines its bindings in the HTML and attaches to existing DOM nodes. While you can hydrate the DOM with Solid it is designed only to work realistically with things previously server-rendered with Solid.

While performance-wise Solid is considerably faster and smaller, Alpine is still better for Alpine's use case.
